demo/src/pages/index.js

```diff
@@ -139,6 +140,9 @@ const IndexPage = () => {
             <li>Final list item</li>
           </ul>
           <h2>Search inputs</h2>
```

Comment:
small thing, but I notice that in the demo this breaks up the `SearchInput` section. You're going to want to move this `h2` to be right next to the `SearchInput`